changelog/fix-7254-update-dispute-fee-text

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Minor wording change on unreleased feature.
```

Comment:
This changelog file will result an entry. I think it should be just a changelog comment.

Reply:
I was just working on another PR and thought I did this one wrong, I checked the diff and it looked great 🤩 so I moved on. 

Turns out I was looking at your changes! TY!